README.md

```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
```

Comment:
I don't know if we can change the title of the ReadMe. It's based on the repo name. 

@narmidm  - can you confirm?